Civi/Api4/Service/Spec/Provider/BatchCreationSpecProvider.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC https://civicrm.org/licensing
+ */
+
+
+namespace Civi\Api4\Service\Spec\Provider;
+
+use Civi\Api4\Service\Spec\FieldSpec;
+use Civi\Api4\Service\Spec\RequestSpec;
+
+class BatchCreationSpecProvider implements Generic\SpecProviderInterface {
+
+  /**
+   * @inheritDoc
+   */
+  public function modifySpec(RequestSpec $spec) {
+    $spec->getFieldByName('created_id')->setDefaultValue('user_contact_id');
+    $spec->getFieldByName('created_date')->setDefaultValue('now');
+    $spec->getFieldByName('modified_id')->setDefaultValue('user_contact_id');
+    $spec->getFieldByName('modified_date')->setDefaultValue('now');
+
+    $entity_table = new FieldSpec('entity_table', 'Batch', 'String');
+    $entity_table->setTitle('Batch Entity Table - remove?');
+    $spec->addFieldSpec($entity_table);
```

Comment:
I don't understand what this extra field is for. I don't see any handling for an `entity_table` param in the `CRM_Batch_BAO_Batch::create()` function.